src/frontend/Komponenter/Personoversikt/BehandlingsoversiktTabell.tsx

```diff
@@ -142,6 +169,21 @@ export const BehandlingsoversiktTabell: React.FC<{
     const finnHenlagtÅrsak = (behandling: BehandlingsoversiktTabellBehandling): string =>
         behandling.henlagtÅrsak ? ` (${henlagtÅrsakTilTekst[behandling.henlagtÅrsak]})` : '';
 
+    const utledBehandlingResultatTilTekst = (behandling: BehandlingsoversiktTabellBehandling) => {
```

Comment:
Denne er kun for applikasjon som ikke er BehandlingApplikasjon.EF_SAK. Altså tilbakekreving og klage. Burde vi kommentere dette i koden. Validere, bedre navn på const, osv?